sequelizeDatabase/sequelFunctions.js

```diff
@@ -403,6 +484,22 @@ retrieveCurrentEmployeesOfCompany = CompanyID => {
   });
 };
 
+retrieveIndividualsByOriginalCompany = companyID => {
```

Comment:
Either use the word "get" or "retrieve" but don't use them both for the same meaning for function names